jpegexport.py

```diff
@@ -72,7 +73,7 @@ def effect(self):
         bgcol = self.options.bgcol
 
         if self.options.page == False:
-            if len(self.selected) == 0:
+            if len(self.svg.selected) == 0:
```

Comment:
This is going to be deprecated *again* in the next version. Just a heads up - will work with 1.0 and 1.1.

Reply:
hey. what will be the replacement in newer InkScape? Can you give some point for this? I am working hard to migrate all my used plugins to python 3 where nobody else did it yet. i dont want to do this twice per year :-D

Reply:
Here's a list of currently known deprecations for 1.1:

https://gitlab.com/inkscape/extensions/-/commit/4838d285b01106d243399e6bf36e7956a291eb98

There's also the bounding box thing among them (but it's for the current selection, so maybe not what you need).